scripts/lib-franklin.js

```diff
@@ -133,6 +133,9 @@ export function decorateIcons(element = document) {
       return;
     }
     const icon = span.classList[1].substring(5);
+    if (span.parentElement.nodeName === 'A' && span.parentElement.ariaLabel === null) {
+      span.parentElement.ariaLabel = icon;
```

Comment:
I'm pasting in Amol's comment since it isn't showing here:

> "We don't need this bit if we have a blanket code that sets the aria-label to the title or the link across the board. No point putting in the icon name for this."